inference/huggingface/zero_inference/run_model.py

```diff
@@ -155,32 +154,25 @@ def run_generation(
     quant_group_size,
     pin_kv_cache,
     async_kv_offload,
+    loops,
 ):
     # Load tokenizer
-    config = get_model_config(model_name)
-    return_token_type_ids = True 
-    padding_side = "left" if config.model_type in ["opt"] else "right"
+    config = get_model_config(model_name)    
 
     if config.model_type == "opt":
         tokenizer = AutoTokenizer.from_pretrained(
             model_name.replace("175b", "66b"), 
-            return_token_type_ids=return_token_type_ids,
-            padding_side=padding_side
+            padding_side="left" 
         )
     else:
-        tokenizer = AutoTokenizer.from_pretrained(
-            model_name,
-            return_token_type_ids=return_token_type_ids,
-            padding_side=padding_side
-        )
-
+        tokenizer = AutoTokenizer.from_pretrained(model_name)
 
     tokenizer.pad_token = tokenizer.eos_token
 
-    if hasattr(config, 'torch_dtype'):
-        dtype = config.torch_dtype
+    if getattr(config, 'torch_dtype', None) is None:
```

Comment:
It looks like dtype is only used in the get_ds_model function to decide the 16-bit type. So perhasp this part of type selection logic can be moved into get_ds_model?

Reply:
Done